.github/workflows/ci-testing.yml

```diff
@@ -19,6 +19,12 @@ jobs:
         os: [ubuntu-latest, macOS-latest, windows-latest]
```

Comment:
[LitGPT's config](https://github.com/Lightning-AI/litgpt/blob/57f0a6105049cee68443ac2f763cad4979807c78/.github/workflows/cpu-tests.yml#L25) looks like following and I would say that we use the same config here too. @lantiga what do you think? 

```
      matrix:
        include:
          - {os: "macOS-12", python-version: "3.10"}
          - {os: "ubuntu-22.04", python-version: "3.11"}
          - {os: "ubuntu-22.04", python-version: "3.10"}
          - {os: "ubuntu-22.04", python-version: "3.9"}
          - {os: "ubuntu-20.04", python-version: "3.8"}
          - {os: "windows-2022", python-version: "3.10"}
```